resources.sql

```diff
@@ -6,17 +6,17 @@ CREATE STAGE IF NOT EXISTS artifacts;
 PUT file://&artifact_name @artifacts AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
 
 CREATE OR REPLACE PROCEDURE HELLO_WORLD_PROC()
-    RETURNS integer
+    RETURNS TABLE(hello_world string)
```

Comment:
This could also be 
```
RETURNS TABLE()
```
if you think that the simplicity and guiding users to realize they don't need to specify all the columns in the table is nice.